evolve_keyboard_layout_wasm/www/app.js

```diff
@@ -316,26 +317,38 @@ Vue.component('evaluator-app', {
             await this.evaluateExisting()
         },
 
-        async updateLayoutConfig (layoutConfig) {
+        async updateLayoutConfig(layoutConfig) {
             this.layoutConfigs[this.selectedLayoutConfig] = layoutConfig
 
             await this.initLayoutEvaluator()
             await this.evaluateExisting()
 
         },
 
-        async selectLayoutConfigType (selectedLayoutConfig) {
+        async selectLayoutConfigType(selectedLayoutConfig) {
             this.selectedLayoutConfig = selectedLayoutConfig
 
             await this.initLayoutEvaluator()
             await this.evaluateExisting()
         },
 
-        removeLayout (layout) {
+        removeLayout(layout) {
             this.details = this.details.filter((d) => d.layout !== layout)
         },
 
-        async optimizeInput () {
+        async startOptimization() {
+            if (this.optMode === "simulated annealing") {
```

Comment:
I prefer having no whitespace in such keys. Could you change that to "simulated_annealing"?